README.cn.md

```diff
@@ -238,7 +238,7 @@ Database Error #2054: The server requested authentication method unknown to the
     +-----------+------------------+-----------------------+
     1 rows in set (0.00 sec)
     ```
-1. 你可能会看到 `caching_sha2_password`。 如果看到了，请运行以下命令：
+1. 你可能会看到 `caching_sha2_password`。 如果是这样，请运行以下命令：
```

Comment:
Change this `1.` to `5.`, as well as the following `1.` to `6.`

Reference: https://www.markdownguide.org/basic-syntax/#ordered-lists